### PR TITLE
fix(DB/Temple of AhnQiraj): Added missing spawns of Cthun Graps.

### LINKED ISSUE #12800: [AQ40] Missing Grasp of C'Thun (180745) objects on top of the 3 questgivers nearby C'Thun's arena.
### Current Behaviour

Gameobjects missing

### Expected Blizzlike Behaviour

Those gameobjects should be there until the encounter with C'Thun is complete.

### Source

Retail

### Steps to reproduce the problem

1. `.go xyz -8674.72 2003 108.90062 531 1`
2. see if the objects are there and prevent the player from interacting with the NPCs
3. complete the C'Thun encounter
4. go back and see if objects have despawned

### Extra Notes

Full spawn

```
[3] UpdateType: CreateObject1
[3] Object Guid: Full: 0x2C31084260B08240000C870000EEE80C GameObject/0 R3138/S3207 Map: 531 Entry: 180745 Low: 15656972
[3] Object Type: 8 (GameObject)
[3] NoBirthAnim: False
[3] EnablePortals: False
[3] PlayHoverAnim: False
[3] HasMovementUpdate: False
[3] HasMovementTransport: False
[3] Stationary: True
[3] HasCombatVictim: False
[3] HasServerTime: False
[3] HasVehicleCreate: False
[3] HasAnimKitCreate: False
[3] HasRotation: True
[3] HasAreaTrigger: False
[3] HasGameObject: False
[3] HasSmoothPhasing: False
[3] ThisIsYou: False
[3] SceneObjCreate: False
[3] HasPlayerCreateData: False
[3] HasConversation: False
[3] PauseTimesCount: 0
[3] Stationary Position: X: -8652.204 Y: 2020.9232 Z: 108.57742
[3] Stationary Orientation: 0.24434558
[3] GameObject Rotation: X: 0 Y: 0 Z: 0.12186909 W: 0.9925462
[3] FieldFlags: 0 (None)
[3] EntryID: 180745
[3] DynamicFlags: 4294901760
[3] Scale: 0.025000000372529029
[3] DisplayID: 5692
[3] SpellVisualID: 0
[3] StateSpellVisualID: 0
[3] SpawnTrackingStateAnimID: 1716
[3] SpawnTrackingStateAnimKitID: 0
[3] StateWorldEffectsQuestObjectiveID: 0
[3] CreatedBy: Full: 0x0
[3] GuildGUID: Full: 0x0
[3] Flags: 0
[3] ParentRotation: X: 0 Y: 0 Z: 0 W: 1
[3] FactionTemplate: 0
[3] State: 1
[3] TypeID: 5
[3] PercentHealth: 255
[3] ArtKit: 0
[3] CustomParam: 0
[3] Level: 0
[3] AnimGroupInstance: 0
```

Spawn Points:

```
[3] Stationary Position: X: -8652.204 Y: 2020.9232 Z: 108.57742
[3] Stationary Orientation: 0.24434558
[3] GameObject Rotation: X: 0 Y: 0 Z: 0.12186909 W: 0.9925462

[1] Stationary Position: X: -8660.628 Y: 2022.3989 Z: 108.57742
[1] Stationary Orientation: 3.6477413
[1] GameObject Rotation: X: 0 Y: 0 Z: -0.9681473 W: 0.2503814

[6] Stationary Position: X: -8663.34 Y: 2029.8976 Z: 108.57742
[6] Stationary Orientation: 4.4505906
[6] GameObject Rotation: X: 0 Y: 0 Z: -0.7933531 W: 0.6087617
```

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/fc1657c910b497f8bf6b1a4e1249156aead3bcaa

### Operating system

Windows 10

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #12800

## Issues Addressed:

- Closes #12800

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.go xyz -8674.72 2003 108.90062 531 1`
See blue circles on the ground

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
The objects are there but I'm sure that they should despawn when C'Thun dies. I think the video you linked had them respawned because the player was, in their words, "borrowing the raid ID". They weren't there when the boss died and it could've respawned due to time or something else.

File: data/sql/updates/pending_db_world/rev_1660996114567482800.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `gameobject` WHERE `id`=180745;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1660996114567482800.sql:2]: delete GUID and ID as GUID is pri

Comment (UltraNix) [reply data/sql/updates/pending_db_world/rev_1660996114567482800.sql:2]: And what's the difference here?

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1660996114567482800.sql:2]: yea but you do not delete the GUID, which can cause SQL errors as GUID is the Primary Key =)

**Comment by UltraNix** [comment]
@Gultask "borrowing the raid ID"? It does not matter. Boss is not respawned.

**Review by Kitzunu — APPROVED** [review]

**Comment by UltraNix** [comment]
@Gultask Despawn mechanic implemented.

**Review by Nyeriah — APPROVED** [review]

**Comment by UltraNix** [comment]
@azerothcore/testers Please, test it.

**Comment by Gultask** [comment]
Sorry for the delay. Doesn't work.

I killed C'Thun but the gameobjects are still there.

**Comment by Gultask** [comment]
It works 👍

**Pull request merged**